app/presenters/base_letter_presenter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class BaseLetterPresenter < BasePresenter
+  include WasteExemptionsEngine::ApplicationHelper
+
+  def contact_full_name
+    format_names(contact_first_name, contact_last_name)
+  end
+
+  def date_of_letter
+    Time.now.to_date.to_formatted_s(:day_month_year)
+  end
+
+  def human_business_type
+    I18n.t("waste_exemptions_engine.pdfs.certificate.busness_types.#{business_type}")
+  end
+
+  def operator_address_one_liner
+    address_lines(operator_address).join(", ")
+  end
+
+  def site_address_one_liner
+    address_lines(site_address).join(", ")
+  end
+
+  private
+
+  def address_lines(address)
+    return [] unless address
+
+    address_fields = %i[organisation premises street_address locality city postcode]
+    address_fields.map { |field| address.public_send(field) }.reject(&:blank?)
```

Comment:
[NIT] 
```suggestion
    address_fields.map { |field| address.public_send(field).presence }.compact
```

Reply:
I get what `presence` and `compact`, but I don't understand why they are better to use than `reject(&:blank?)`.

The key point for me is that the intent is more meaningful using `reject(&:blank?)` whereas I had to go google what presence and compact do. However this is 100% my own opinion. The rest of the team might feel different.

Reply:
This is why this is an NIT. Me, I have used `presence` and `compact` so many times that I feel that's more clear, this is more of a "this is another way of achieving this", but not necessarily a better one :)

Reply:
`reject(&:blank)` is what I'm most used to and I think we've used it fairly consistently around the various codebases, so I'd rather stick with that. But useful to know about alternatives 👍

Reply:
Sounds good! :D